user/network-topology.rst

```diff
@@ -122,6 +122,101 @@ Sending data for topology with RECEIVE strategy
    or, alternatively, a non-admin visualizer page is also available at
    the URL ``/topology/topology/<TOPOLOGY-UUID>/``.
 
+Sending data for ZeroTier topology with RECEIVE strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Follow the procedure described below to
+setup **ZeroTier** topology with **RECEIVE** strategy.
+
+**Note:** In this example, the **Shared
+systemwide (no organization)** option is used
+for the ZeroTier topology organization. You are free
+to opt for any organization, as long as both the topology
+and the device share the same organization, assuming the
+`OpenWISP controller integration <#integration-with-openwisp
+-controller-and-openwisp-monitoring>`_ feature is enabled.
+
+1. Create topology for ZeroTier
+###############################
+
+1. Visit ``admin/topology/topology/add`` to add a new topology.
+
+2. We will set the **Label** of this topology to ``ZeroTier`` and
+   select the topology **Format** from the dropdown as ``ZeroTier``.
+
+3. Select the strategy as ``RECEIVE`` from the dropdown.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-network-topology/docs/docs/zerotier-tutorial/topology-1.png
+   :alt: ZeroTier topology configuration example 1
+
+4. Let use default **Expiration time** ``0``
+   and make sure **Published** option is checked.
+
+5. After clicking on the **Save and continue editing** button,
+   a topology receive URL is generated. Make sure you copy
+   that URL for later use in the topology script.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-network-topology/docs/docs/zerotier-tutorial/topology-2.png
+   :alt: ZeroTier topology configuration example 2
+
+2. Create a script for sending ZeroTier topology data
+#####################################################
+
+1. Now, create a script (e.g: ``/opt/send-zt-topology.sh``)
```

Comment:
When following guides on OpenWRT related things, when I encounter something like "Create a script", the first question I have is where...
My laptop, the openwrt device or server.

Can you please mention that here?

Reply:
Sure!